prover/Makefile

```diff
@@ -3,10 +3,10 @@ all:
 	make tendermint-sweep
 	make fibonacci-groth16
 
-build-groth16:
+build-plonk-bn254:
 	mkdir -p build && \
 	RUSTFLAGS='-C target-cpu=native' \
-	cargo run -p sp1-prover --release --bin build_groth16 -- \
```

Comment:
Also in this file, given that we no longer use `tendermint-sweep`, `fibonacci-groth16` or `fibonacci-sweep` in testing, the default commands here should be `build-plonk-bn254` + `release-plonk-bn254`. Then users just need to run `make` in the `prover` directory.